dspace-api/src/main/java/org/dspace/content/RelationshipTypeServiceImpl.java

```diff
@@ -84,4 +84,8 @@ public void delete(Context context,RelationshipType relationshipType) throws SQL
         }
         relationshipTypeDAO.delete(context, relationshipType);
     }
+
+    public List<RelationshipType> findByLeftOrRightLabel(Context context, String label) throws SQLException {
```

Comment:
Please add Javadocs

Reply:
Since this an Implementation class, does it suffice to have the JavaDocs in the Interface (similar to other Implementation classes)?

Reply:
@benbosman : yes, that's fine, but then we need an `@Overrides` annotation on this method. That makes it clear that the JavaDocs are on the interface.